### PR TITLE
Updating variables

### PR DESCRIPTION
**Pull request by jgod1360**
https://github.ibm.com/IBM-DEG/Ecosystem-Technical-Enablement/issues/288

### CODE REVIEW

File: terraform/cp4ba/variables.tf [hunk 1]
```diff
@@ -70,7 +70,7 @@ variable "owner" {
 
 variable "cluster_config_path" {
   default     = "./.kube/config"
```
Comment (annumberhocker) [inline terraform/cp4ba/variables.tf:72]: Why did you remove the comment regarding Schematics?  This is still necessary as far as I know.

**Review by annumberhocker — CHANGES_REQUESTED** [review]
Just one small update to the description for kubeconfig.

File: terraform/cp4ba/variables.tf (hunk 1 above)
Comment (jgod1360) [reply terraform/cp4ba/variables.tf]: Done.
```
  description = "directory to store the kubeconfig file. If running on Schematics, use `/tmp/.schematics/.kube/config`"
```

**Review by annumberhocker — APPROVED** [review]
LGTM
